suite/tests/client-interface/mangle_suspend.c

```diff
@@ -60,22 +63,28 @@ suspend_thread_routine(void *arg)
     /* This thread is executing labels for the client to insert a clean call that
      * does the suspend and subsequent check for correctness.
      */
+    int suspend_val = *(int *)arg;
     while (!test_ready) {
         /* Empty. */
     }
-    struct timespec sleeptime;
-    sleeptime.tv_sec = 0;
-    /* Pick an uneven number to maximize the number of suspend calls w/o
-     * running into limitations in DR.
```

Comment:
What limitations?  Is there some bug somewhere to xref?

Reply:
Just filed i#3332.